src/files.h

```diff
@@ -140,7 +140,8 @@ R"RAW(
 #endif
 R"RAW(
         "close_tab": "<primary>w",
-        "window_toggle_split": ""
+        "window_toggle_split": "",
+        "clear_console": "<primary><shift>c"
```

Comment:
When increasing the version number, an older preference file gets automatically updated. That is, when someone downloads the latest changes and there are new preference items available, or old ones have been removed (or renamed).